Adds Gas Mask Variety

Paxilmaniac opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds three variants of the gas mask, replaces fire locker gas masks with a random pick between the three new variants, and gives science lockers and vendors one of the new variants in place of the old one.

![image](https://user-images.githubusercontent.com/57122098/72036756-b4c80000-3250-11ea-93ca-e46ad4e19fab.png)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Gas masks are a go to mask, because they both hide your identity AND can be used as internals. This doesn't remove the old gas masks, but makes them much harder to find, and replaces the most common sources with one of three new variants that makes trade offs between eachother, so there are less of the all powerful greytide masks everywhere

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: three gas mask variants
tweak: fire locker gas masks are chosen randomly between these three masks
tweak: science lockers and vendors get the face revealing mask that can use internals
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

carda-sometimesknownas-paxilmaniac committed:
adds da masks

adds three gas mask variations and randomizes fire locker masks

Fikou commented on code/controllers/subsystem/traumas.dm:
```diff
@@ -69,7 +69,8 @@ SUBSYSTEM_DEF(traumas)
 
 					  "greytide"  = typecacheof(list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/cattleprod,
-												 /obj/item/twohanded/spear, /obj/item/clothing/mask/gas)),
+												 /obj/item/twohanded/spear, /obj/item/clothing/mask/gas, /obj/item/clothing/mask/gas/alt, /obj/item/clothing/mask/gas/alt2,
```
they dont strike the same fear as the old one

Paxilmaniac replied:
ok good point

carda-sometimesknownas-paxilmaniac committed:
the new masks aren't as scary to those scared of the tide

new masks aren't a part of the og tider outfit so they no longer trigger the trauma

carda-sometimesknownas-paxilmaniac committed:
RETURN OF THE ROGUE PIXEL

HOW DID IT EVEN GET ON THE ATMOS GAS MASK I NEVER TOUCHED IT

wesoda25 commented:
I mean I guess

cacogen commented:
I like the new? atmos gas mask, it looks menacing like New Vegas and is a refreshing change to that old tired sprite. I don't like these sprites though. If you just made recolours of the existing sprite in the same vein as that i.e. lenses and/or the gas mask itself that'd be cool

itseasytosee commented:
Halp grey man with mask robust me?? I do not know who he is?? remobe bad mask!

Paxilmaniac commented:
it doesn't remove the gas mask we already have, it just makes them harder to get than just "hmm yes red locker in hall"

Twaticus commented:
did you mean to remove the atmos gas mask mob sprite or?

Paxilmaniac commented:
in case it isn't clear: nothing is removed in this pr, it adds three masks that spawn in place of the current gas masks in the red lockers around the hallways.

this doesn't touch the atmos gas mask at all

Paxilmaniac commented:
also the mob sprite for atmos gas masks shouldn't be removed?

Jalleo commented on code/game/machinery/gulag_teleporter.dm:
```diff
@@ -30,7 +30,9 @@ The console is located at computer/gulag_teleporter.dm
-		/obj/item/clothing/mask/gas))
+		/obj/item/clothing/mask/gas,
+		/obj/item/clothing/mask/gas/alt,
+		/obj/item/clothing/mask/gas/alt2))
```
This could cause issues did you test this?

trollbreeder commented:
I like these sprites, especially the shielded visor gas mask.

Mickyan commented:
You need internals for fighting fires, would rather not have to play the rng lottery during an emergency

Personally I'd just swap them out for the regular breath mask but hey

Paxilmaniac commented:
only one of the three cant use internals and its rare to get it, and you have a breath mask in your box, and there are some in every emergency locker, and emergency lockers happen to be almost always right next to a fire locker, so it shouldn't be a problem

ExcessiveUseOfCobblestone commented:
![image](https://user-images.githubusercontent.com/11748095/72026499-a2969380-3249-11ea-8726-157fa25e2335.png)
?

Paxilmaniac commented:
gas_atmos-2 was a mistake i accidentally put in which was literally an accidental copy paste of the atmos gas mask mob sprite with a rogue pixel, if you check the current mob mask sprites the atmos mask sprite is still there
![image](https://user-images.githubusercontent.com/57122098/72027013-5a1fab80-3232-11ea-98d3-44948b490908.png)

REMINDER TO NOT CODE AT THREE AM OK??

wesoda25 commented:
Why do we need it though. Does anyone even complain about this stuff?

Paxilmaniac commented:
i remember people talking about it a while back,

![image](https://user-images.githubusercontent.com/57122098/72027466-ff874f00-3233-11ea-8e68-f1007c837acc.png)

wesoda25 commented:
That comment doesn't even make sense though, how does it make sense to make gas masks, the thing you are supposed to have when fighting fire, not be able to be connected to internals? I don't even get whats "good" about the mask, its a liable reason to be arrested by shitsec so any real advantage it provides is ???? This PR seems like pointless bloat tbh

JMoldy commented:
Can we have literally any gas mask actually function as a gas mask? It's just weird that NONE of the **four** gas masks on the station would be able to actually **work as a gas mask**.

Paxilmaniac commented:
im going to make the 3rd and 4th gas masks actually look different from the original

carda-sometimesknownas-paxilmaniac committed:
masks that i actually spent time on

i only got yelled at for an hour by doot ok

carlarctg commented:
conflict

the mask with  big visor looks really goofy. like it looks dumb. like the big head looks bubbly and it dumb

anconfuzedrock commented:
imagine getting a fucking mask that doesn't even let you breathe from a fire closet in an emergency. Why's the third actually exist? masks that conceal your identity but don't let you breathe are super easy- you can legit make a paper bag mask even. not that anything's hard especially on stations like meta and delta where there's a fuckin autodrobe in public.

carda-sometimesknownas-paxilmaniac committed:
probably fixes the merge conflict also why can commit titles be beeeeeeg

the title be spitting facts though

carda-sometimesknownas-paxilmaniac committed:
ok we like merge conflicts

im just following the orang man guide ok

carda-sometimesknownas-paxilmaniac committed:
Revert "ok we like merge conflicts"

This reverts commit 0c5fe762da8370b5d8fd55cf8a919b1cc90e6ac9.

Paxilmaniac commented:
mega oopsie, and im too lazy to fix it

Paxilmaniac commented:
actually get this i'm just going to stop because i seem to have completely fucked it up
